src/osx/checklst_osx.cpp

```diff
@@ -63,7 +63,7 @@ bool wxCheckListBox::Create(
     if ( !wxCheckListBoxBase::Create( parent, id, pos, size, n, choices, style & ~(wxHSCROLL | wxVSCROLL), validator, name ) )
         return false;
 
-    int colwidth = 30;
+    int colwidth = 18;
```

Comment:
Just curious, where does this come from?

Reply:
I'm afraid it's the same lazy magic number as 30 was. 16pt looked too compressed, 18pt looked fine. It would be best to calculate this somehow from intrinsic size, but I'm not sure how and figured this is better than nothing.